x/stake/keeper/distribute.go

```diff
@@ -0,0 +1,127 @@
+package keeper
+
+import (
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/stake/types"
+)
+
+func (k Keeper) DistributeBeaconChainInBreathBlock(ctx sdk.Context, bscFee int64) {
+	for k.hasNextBatchRewards(ctx) {
+		k.distributeSingleBatch(ctx, MockSideChainIDForBeaconChain)
+	}
+
+	validators, height, found := k.GetHeightValidatorsByIndex(ctx, daysBackwardForValidatorSnapshot)
```

Comment:
`daysBackwardForValidatorSnapshot` for BNB Smart Chain is 3, while for Beacon Chain I think it is 1.

Reply:
Since the validator set update strategy of BC has been aligned with BSC, here we should use 3.
Do you mean we didn't save them in snapshot for 3 days?